ru/mail/branding/nightly/brand.dtd

```diff
@@ -1,3 +1,3 @@
-<!ENTITY  brandShortName        "Postbox">
+﻿<!ENTITY  brandShortName        "Postbox">
 <!ENTITY  brandFullName         "Postbox">
 <!ENTITY  vendorShortName       "">
```

Comment:
Why? Please explain.
Please see ...\en-GB\mail\chrome\branding\brand.dtd as refrence. This is not a copyright statement..